gym/envs/registration.py

```diff
@@ -316,8 +329,10 @@ def make(self, path: str, **kwargs) -> Env:
 
         # We check what the latest version of the environment is and display a warning
         # if the user is attempting to initialize an older version.
-        latest_version, latest_ns = max(versions)
-        if int(version) < latest_version:
+        is_lesser_version, latest_version, latest_ns = self._is_lesser_version(
```

Comment:
Instead of `_is_lesser_version` can we do something like,

```py
from operator import itemgetter

...

latest_version, latest_ns = max(filter(itemgetter(0), versions), default=(None, None,))
if version and latest_version and int(version) < latest_version:
    ...
```

I think it's clear what's going on with the one-liner solution, we filter by the first element, if it's None it won't be included. If we perform `max([])` the default kicks in and will return `(None, None)`.

Reply:
Right, that was probably overkill.